### PR TITLE
Add provisional Ignite support

### PR DESCRIPTION
**Pull request by luxas**
@robertojrojas will make this provisional work better, and submit a better version of this in a future PR :)

### CODE REVIEW

File: pkg/cluster/cluster.go [hunk 1]
```diff
@@ -174,44 +224,74 @@ func (c *Cluster) createMachine(machine *Machine, i int) error {
...
-	}
-	publicKey, err := c.publicKey()
-	if err != nil {
-		return err
-	}
-	if err := copy(name, publicKey, "/root/.ssh/authorized_keys"); err != nil {
-		return err
+		// We connect the first network in with the run command, connect the remaining
+		// ones.
+		// TODO(damien): Split run into create+start so we can connect all networks
```
Comment (dlespiau) [inline pkg/cluster/cluster.go]: This comment is now out of date and should be removed.

File: pkg/cluster/cluster.go [hunk 2]
```diff
@@ -153,6 +156,48 @@ mkdir $sshdir; chmod 700 $sshdir
...
+			exitCode = exitError.ExitCode()
+		} else {
+			cmdErr = fmt.Errorf("failed to get exit code for external command %q", cmdArgs)
+		}
+	}
+
+	return exitCode, cmdErr
+}
+
+/*func igniteImageName(dockerImgName string) (string, error) {
```
Comment (dlespiau) [inline pkg/cluster/cluster.go]: A bit of dead code to remove.

File: pkg/cluster/cluster.go [hunk 3]
```diff
@@ -153,6 +156,48 @@ mkdir $sshdir; chmod 700 $sshdir
 touch $sshdir/authorized_keys; chmod 600 $sshdir/authorized_keys
 `
 
+func executeCommand(command string, args ...string) (string, error) {
+	cmd := exec2.Command(command, args...)
+	out, err := cmd.CombinedOutput()
+	cmdArgs := strings.Join(cmd.Args, " ")
+	//log.Debugf("Command %q returned %q\n", cmdArgs, out)
```
Comment (dlespiau) [inline pkg/cluster/cluster.go]: A bit of debugging code to remove.

**Comment by luxas** [comment]
This was fixed with #178; thx @robertojrojas!

**Pull request closed without merging**